keyboards/handwired/hacked_motospeed/readme.md

```diff
@@ -19,5 +19,6 @@ Hardware Availability: Ebay, AliExpress
 Make example for this keyboard (after setting up your build environment):
 
     make handwired/hacked_motospeed:default
+    make handwired/hacked_motospeed:german
```

Comment:
```suggestion
```

This section is just a example, generally it should not list every keymap available.